hack/custom.config.yaml.in

```diff
@@ -31,3 +31,13 @@
     labels:
       suite: cryostat
       test: cryostat-recording
+- op:add
```

Comment:
I think this needs a space between `op:` and `add`, and then a `make bundle`. I was getting an error when running `make bundle`.